src/scss/utils/functions.scss

```diff
@@ -0,0 +1,86 @@
+// Bootstrap functions
```

Comment:
Good question! We need to add the watea variables file before pulling in the starter theme, and all of its bootstrap dependencies. However, the variables file relies on some of the bootstrap functions. I'll have a think about this - maybe there is a better way of handling this.

Reply:
I've refactored this quite a bit having had a chat with Michael so that this can be build independently of the starter theme and removed the copied functions scss file.

Reply:
Seems like it building well no breaking issues I can see. I'm thinking that is what the funny paths were in the `main.scss` file?